Add list of contributors

jxtx opened the pull request.

jxtx committed:
Add list of contributors

dannon commented on LICENSE.txt:
```diff
@@ -13,7 +13,7 @@ Licensed under the Academic Free License version 3.0
        the Original Work;
 
-    c) to distribute or communicate copies of the Original Work and Derivative 
+    c) to distribute or communicate copies of the Original Work aGnd Derivative 
```
s/aGnd/and

nekrut commented:
+1, but I will need to add Tobacco Funds and Huck Funds. I will issue a separate PR for this

natefoo commented:
Many of these email addresses are now invalid, not sure if we should include them at all?

martenson committed:
update my contact

jxtx commented:
I like including them since they are used in the commit messages. Happy for people to update with their current address but I'd still like to keep the old ones.

jmchilton commented:
I think this pull request should probably follow the procedures laid out for governance changes introduced in #314 - though I am absolutely fine if it doesn't - we made the process very onerous.

natefoo commented:
@jmchilton Is this a governance change? Just recognition for past committers, right?

jmchilton commented:
We are clarifying the license also - #314 mentions clarifications specifically.

jxtx commented:
I think #314 only mentions clarification of procedures. No procedure changes here. I would like for #314 not to apply for this one.

jmchilton commented:
@jxtx No objections here then :).

dannon commented:
+1, and I also don't think this is a procedural change.

jxtx committed:
Merge pull request #3 from martenson/patch-1

Update contact info for Marten.

natefoo committed:
Fix Guru's name in CONTRIBUTORS.md

Apparently I messed it up during the hg->git conversion, I'll have to see if I can fix that. =(

jxtx committed:
Merge pull request #4 from natefoo/patch-1

Fix Guru's name in CONTRIBUTORS.md

jmchilton commented:
I am -0 on the current "Institutional sponsors" language - feel free to merge without winning me over on it obviously. No need to let me bite the hand that feeds me :).

More specifically, I don't mind the idea of calling out specific grants that have funded Galaxy core development explicitly, but by introducing it via the institutions it makes it sound for instance as though more Galaxy core development has occurred by employees of say George Washington University _while they were employed there_ than employees of the University of Freiburg _while they were employed there_.

My preference would be something more like the following - which still calls out the same four institutions for their leadership role - but which also makes a more clear distinction I believe between leadership and contribution:

---

# History

Galaxy development began at The Pennsylvania State University in 2006. In 2009 all contributions to that point were licensed by The Pennsylvania State University under the terms of the Academic Free License 3.0 (see LICENSE.txt). This license applies to all subsequent contributions as well. Galaxy development has continued in part due the grants and institutions listed below.

# Grant Sponsorship

Development of the core Galaxy project has been explicitly funded by the following grants.
- NSF DBI 0543285, “Tailoring genomic data to the needs of experimental biologists and educators”
- NIH R21 HG005133, “A turnkey solution for next generation sequence data analysis”
- NIH R01 HG004909, “An efficient lightweight environment for biomedical computation”
- NSF DBI 0850103, “Cyberinfrastructure for accessible and reproducible research in life sciences”
- NIH RC2 HG005542, “Dynamically scalable accessible analysis for next generation sequence data”
- NIH U41 HG006620, “Democratization of data analysis in life sciences through Galaxy”

# Institutional Sponsorship

The following institutions have significantly contributed to the development of the core Galaxy project (those marked with \* through the grants explicitly outlined above).
- The Pennsylvania State University*
- Emory University*
- Johns Hopkins University*
- George Washington University*
- University of Freiburg
- University of Minnesota
- Texas A&M University

martenson commented:
+1 @jmchilton's approach
Should the institutions list correspond to the commiters group?

natefoo commented:
At least in the case of Penn State, there were funds (and not just salary funds) contributed, so perhaps that's what @jxtx was driving at in that section?

jmchilton committed:
Tweak institutional support language in CONTRIBUTORS.md.

jxtx committed:
Merge pull request #5 from jmchilton/contributors-institutions

Tweak institutional support language in CONTRIBUTORS.md.

jmchilton commented:
I go from -0 to +0 for the language on institutional support if the language is modified corresponding to https://github.com/jxtx/galaxy/pull/5.

So I am :+1: on the pull request now.

dannon commented:
Still +1.

nekrut commented:
+1

bgruening commented:
If I'm counting correctly we have more than enough votes to merge this.
Speaking as someone from the community I think we all appreciate this PR (and the coming changes) very much!

Thanks for moving the entire project in this direction. ❤️

mr-c commented:
FYI, here is how I harvested the list of contributors from Git including name & email address cleanup using the `.mailmap` feature of `git log` https://github.com/dib-lab/khmer/pull/1023

In this case I'm outputting LaTeX for our software paper; should be easy to adjust for your own needs.
